Syndication/AzureStack.MarketplaceSyndication.psm1

```diff
@@ -19,6 +19,9 @@ function Export-AzSOfflineMarketplaceItem {
         [ValidateNotNullorEmpty()]
         [String] $ResourceGroup = "azurestack",
 
+        [Parameter(Mandatory = $false, ParameterSetName = 'SyncOfflineAzsMarketplaceItem')]
+        [Switch] $ReduceDownloadThreads = $false,
```

Comment:
AzCopy allows you to explicitly set number of concurrent operations. I would suggest using a number instead of flag.

Reply:
changed